recipes/cgat-apps/meta.yaml

```diff
@@ -32,31 +33,29 @@ requirements:
   run:
     - python
     - cgatcore
-    - alignlib-lite 
-    - biopython 
-    - cython 0.29.*
-    - future 
-    - matplotlib 
-    - numpy 
-    - pandas 
-    - pybedtools 
-    - pybigwig 
-    - pysam 
-    - python-lzo 
-    - pyyaml 
+    - alignlib-lite
+    - biopython
+    - future
+    - matplotlib
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - pybedtools
+    - pybigwig
+    - {{ pin_compatible('pysam') }}
```

Comment:
If you're using cython to compile against pysam you probably need to add `max_pin='x.x.x'`, since releases aren't likely to be binary compatible.